.coveragerc

```diff
@@ -15,7 +13,6 @@ plugins = Cython.Coverage
 [paths]
 source =
     lib/cartopy
-    /home/travis/miniconda/envs/test-environment/lib/python?.?/site-packages/cartopy
```

Comment:
This is here to alias (non-editable) installs for coverage purposes. If we only do editable installs on CI and it all works, we can probably get rid of the whole section?

Reply:
:+1: Done.